cmd/faktory-cli/repl.go

```diff
@@ -56,9 +56,7 @@ func repl(path string, store storage.Store) {
 		if er != nil {
 			if io.EOF == er {
 				fmt.Println("")
-				if store != nil {
```

Comment:
This is a regression, was recently fixed to make this check.

Reply:
Good call, think I must have had an old version of the file opened that overwrote this change.